packages/terra-dev-site/src/layouts/_DevSiteNavigationLayout.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { useLocation, useHistory, useRouteMatch } from 'react-router-dom';
+// import IconSearch from 'terra-icon/lib/icon/IconSearch';
+// import IconTile from 'terra-icon/lib/icon/IconTile';
+import PrimaryNavigationLayout, { NavigationItem } from '@cerner/terra-application/lib/primary-navigation-layout';
+import { PageContainer } from '../terra-application-temporary/page';
+
+import DevSitePage from '../pages/_DevSitePage';
+import NotFoundPage from '../pages/_NotFoundPage';
+// import SettingsModal from '../modals/_SettingsModal';
+// import SearchModal from '../modals/_SearchModal';
+// import ApplicationSwitcherModal from '../modals/_ApplicataionSwitcherModal';
```

Comment:
Modals have been temporarily removed from the application. The new Modals require changes in terra-application base and cannot be patched in similarly to how I added pages.